chef/data_bags/crowbar/bc-template-provisioner.json

```diff
@@ -102,7 +102,7 @@
     "provisioner": {
       "crowbar-revision": 0,
       "crowbar-applied": false,
-      "schema-revision": 20,
+      "schema-revision": 21,
       "element_states": {
```

Comment:
We need to update this to 22 now. (Since https://github.com/crowbar/barclamp-provisioner/pull/385 was merged.)

Though I wonder why github stil thinks this is mergable.